WooCommerce/Classes/ViewRelated/Custom Fields/CustomFieldsDetails.swift

```diff
@@ -111,13 +182,25 @@ private extension CustomFieldRow {
     }
 }
 
+
+private extension EditableCustomFieldRow {
+    enum Constants {
+        static let spacing: CGFloat = 8
+        static let vStackPadding: CGFloat = 16
+        static let hStackPadding: CGFloat = 10
+        static let height: CGFloat = 64
+    }
+}
```

Comment:
Nit: instead of duplicating the constants, you can bring the enum to `CustomFieldsDetails` extension and reuse it for both `CustomFieldRow` and `EditableCustomFieldRow`.

Reply:
Updated as part of https://github.com/woocommerce/woocommerce-ios/pull/13639/commits/1706a3fa8a3a6a4e9abe47892321f5971b050da2